.github/create_templates.ts

```diff
@@ -82,6 +84,7 @@ const templates: Record<string, Answers> = {
 		indentation: "Tab",
 		quotes: "double",
 		adminReact: "yes",
+		tabReact: "yes",
```

Comment:
Should we really add a tab to every template by default?
Most adapters don't need or use it.

Reply:
Good question, fact is, it is already now enabled in the default config (see line 47). The only thing I did, is make sure if we have React for Admin, we also use React for the Tab.

Reply:
You're right, I forgot about that.